### PR TITLE
[Feat] list component

### PR DESCRIPTION
**Pull request by rushikesh3299**
Ordered lists
Unordered list
Notification list

Animation

### CODE REVIEW

**Review by Bharatrathod-81 — APPROVED** [review]
good to go

File: Components/Lists/list.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  list-style-type: lower-roman;
+}
+
+.notification-list {
+  list-style-type: none;
+  width: 50%;
+  padding-left: 0;
+}
+
+@media screen and (max-width: 850px) {
```
Comment (ajit-sharmaGH) [inline Components/Lists/list.css:27]: I think you missed to shift media queries at the bottom of the file

File: Components/Lists/list.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  background-color: var(--bg-white);
+  display: flex;
+  gap: 2rem;
+}
+
+.notification-list img {
+  margin-left: 0.5rem;
+}
+
+.notification-list-text :nth-child(2) {
```
Comment (ajit-sharmaGH) [inline Components/Lists/list.css:46]: Good naming convention

**Review by ajit-sharmaGH — APPROVED** [review]
everything looks fine 
You can merge it

**Pull request merged**